index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link href="https://fonts.googleapis.com/css?family=Poppins:400,600" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <Header>
+        <div class="logo">My Logo</div>
+        <div class="menu"><img src="images/Union.png" alt="menu"></div>
```

Comment:
- [ ] the Hamburger menu should be in a `nav` tag see the screenshot 👇
![nav](https://user-images.githubusercontent.com/61562626/129187260-d0ccbe27-f0db-4d0d-9aa4-94a819b9f8a6.jpeg)